.github/workflows/create-vatz-bi-weekly.yml

```diff
@@ -0,0 +1,65 @@
+name: Create a next vatz discussion & milestone
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: "0 13 * * 3" # Every Wednesday at 1 p.m.
+env:
+  URL: https://api.github.com/repos/dsrvlabs/vatz/milestones # NEED TO CHANGE
+jobs:
+  create-bi-weekly:
```

Comment:
Please, add some comment for each step that's going to help a lot for everyone to understand.

Reply:
> commit: 973154dfe649d307c5bfdb56995bd1662fd5e781

Add comment!